wp-content/themes/redmond/views/layout/footer.html.twig

```diff
@@ -1,3 +1,37 @@
 <footer class="footer">
-
-</footer>
\ No newline at end of file
+    <div class="row top-row">
+        {% for column in SP.ACF.get_field('footer_icons', 'option') %}
+            <div class="large-4 medium-4 columns icon-title">
+                <a href="#"><img class="footer-icon" src="{{ column.icon.url }}" /></a>
+                <h2><small>{{ column.small_header }}</small> {{ column.large_header }}</h2>
+            </div>
+        {% endfor %}
+    </div>
+    <div class="row">
```

Comment:
since the other two parts are named you probably want to name this one too. maybe footer-content or footer-widgets